### PR TITLE
Create option --without-single

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

**Comment by dpo** [comment]
Thanks. The test on line 143 still runs in single precision (and will fail if `--without-single` is used).

I suppose the SIF decoder should also have the same option.

**Comment by abelsiqueira** [comment]
Line 143 is fixed. And travis was updated to run both single and both precisions.

The sifdecoder doesn't seem to have anything related to single precision on linux at least.

File: .travis.yml [hunk 1]
```diff
@@ -20,12 +20,16 @@ install:
...
+  - PACKAGE=mastsif SINGLE="--without-single"
+  - PACKAGE=archdefs SINGLE=""
+  - PACKAGE=sifdecode SINGLE=""
+  - PACKAGE=cutest SINGLE=""
+  - PACKAGE=mastsif SINGLE=""
 
 script:
   #- brew audit $PACKAGE --strict  # Fails for now b/c head only.
-  - brew install -v $PACKAGE --HEAD
+  - brew install -v $PACKAGE $SINGLE --HEAD
```
Comment (dpo) [inline .travis.yml]: Is this necessary? Omitting `--without-single` builds both double and single precision, so we'll know what builds and what doesn't.

File: cutest.rb [hunk 2]
```diff
@@ -135,7 +140,9 @@ def caveats
     cd testpath do
       %w[gen77 gen90 genc].each do |pkg|
         system "runcutest", "-p", pkg, "-D", "#{libexec}/sif/ROSENBR.SIF"
-        system "runcutest", "-p", pkg, "-sp", "-D", "#{libexec}/sif/ROSENBR.SIF"
+        if build.with? "single"
+          system "runcutest", "-p", pkg, "-sp", "-D", "#{libexec}/sif/ROSENBR.SIF"
+        end
```
Comment (dpo) [inline cutest.rb]: Sorry to bother, but this should be a one-liner:
```ruby
system "runcutest", "-p", pkg, "-sp", "-D", "#{libexec}/sif/ROSENBR.SIF" if build.with? "single"
```

**Comment by dpo** [comment]
Thanks for your patience ;).

File: cutest.rb (hunk 2 above)
Comment (abelsiqueira) [inline cutest.rb]: Sorry, ruby isn't a first language.

**Comment by abelsiqueira** [comment]
Fixed. Removed last commit and fixed the other.

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**